tests/security/cc/audit_remote.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Run CC 'audit-remote test case'
```

Comment:
Does it can be,
Run CC **'audit-remote'** test case ?

Reply:
Fixed. Thx